packages/core/src/external/carequality/ihe-gateway-v2/saml/security/security-header.ts

```diff
@@ -122,12 +138,10 @@ export function createSecurityHeader({
             },
             {
               "@_Name": "urn:oasis:names:tc:xacml:2.0:subject:role",
-              "@_NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:uri",
```

Comment:
double checking that we don't need the format here anymore?

Reply:
yes not needed / caused one or two endpoints to error